TicketUtil.py

```diff
@@ -1,6 +1,5 @@
 import logging
 import re
-
```

Comment:
Keep this blank line here according to PEP8: https://www.python.org/dev/peps/pep-0008/#imports